bin/kano-uixinit

```diff
@@ -79,6 +102,8 @@ set_keyboard $kano_keyboard
 # Jump to Dashboard Mode, unless instructed otherwise
 if [ "${DESKTOP_MODE}" != "1" ]; then
     logger --id --tag "info" "kano-uixinit terminates into Dashboard Supervisor"
+    # only run updater once each boot
+    [ "$DESKTOP_RUN" -eq 1 ]  &&  sudo kano-updater ui boot-window
```

Comment:
ah, in that case the hourglass would timeout after 20 seconds, @Ealdwulf . Not such a good idea unless we'd have some kind of window to wait for.